components/product-page/product-page.css

```diff
@@ -0,0 +1,155 @@
+@import url(/style.css);
+@import url(/components/color.css);
+:root{
+    --white-less:rgb(253, 249, 249);
+}
+
+.main-body{
+    margin-top:7rem;
+}
+.header-nav{
+    position: fixed;
+    z-index:2;
+    top:0;
+    width:100%;
+    box-shadow: 0px 15px 10px -15px #111;  
+
+}
+
+.product-page{
+    display:flex;
+    flex-wrap:wrap;
+    gap:3rem;
+    justify-content: center;
+    margin-left:23rem;
+}
+
+.side-drawer{
+    position: fixed;
+    z-index:444;
+    background-color: var(--white-less);
+    height:100%;
+    font-size:1.2rem;
```

Comment:
give header nav greater z-index value than side-drawer so the navbar shadow looks consistent across the page.

Reply:
done